ios/Configuration.storekit

```diff
@@ -0,0 +1,53 @@
+{
+  "identifier" : "78370A4E",
+  "nonRenewingSubscriptions" : [
+
+  ],
+  "products" : [
+
+  ],
+  "settings" : {
+
+  },
+  "subscriptionGroups" : [
+    {
+      "id" : "7A3D81C8",
+      "localizations" : [
+
+      ],
+      "name" : "Pro",
+      "subscriptions" : [
+        {
+          "adHocOffers" : [
+
+          ],
+          "displayPrice" : "4.99",
+          "familyShareable" : false,
+          "groupNumber" : 1,
+          "internalID" : "A2FD1DA9",
+          "introductoryOffer" : {
+            "internalID" : "ADF6914F",
+            "paymentMode" : "free",
+            "subscriptionPeriod" : "P1W"
+          },
+          "localizations" : [
+            {
+              "description" : "",
+              "displayName" : "",
+              "locale" : "en_US"
+            }
+          ],
+          "productID" : "pro1gb",
+          "recurringSubscriptionPeriod" : "P1M",
+          "referenceName" : "Pro (1 GB)",
```

Comment:
can we change this to drop the (1 GB) from the name? i'm thinking we should just not have it at first and if we want to make products with more than 1 GB or whatever we can add that clarification to those ones

Reply:
Need to change it on the AppStore as well, but sure.